docs/macros.md

```diff
@@ -173,6 +173,18 @@ applies the weird escape sequences required for bigquery and snowflake
 **Returns**: 
 ##### Supports: _Postgres, Snowflake, BigQuery, Redshift_
 ----
+### [array_contains](../macros/array_contains.sql)
```

Comment:
Theoretically, this should have been updated after the PRs for it were merged. It looks like the `commit docs` step is actually failing. Added issue for it: #67 

Thanks for including the updated docs!

Reply:
Ok, good to know. I was unsure whether to include it or not...